addons/addon-base-raas/packages/base-raas-services/lib/schema/update-aws-accounts.json

```diff
@@ -26,7 +26,7 @@
       "type": "string",
       "pattern": "^[0-9]{12}$"
     },
-    "cfnStackName": {
+    "cfnStackId": {
       "type": "string",
```

Comment:
Add regex validation for stack arn?

Reply:
got it 👍🏻. Some of the other schemas didn't do any validation for the ARNs aside from this, but it makes a lot more sense to add it.